cuqi/distribution/_truncated_normal.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from scipy.special import erf
+from cuqi.distribution import Distribution
+
+class TruncatedNormal(Distribution):
```

Comment:
You made this really nice showcases in the PR @chaozg. I suggest you add one or two of them as "Examples" in the docstring!

Reply:
Added a example in the docstring as suggested